forge-gui/res/cardsfolder/upcoming/banes_contingency.txt

```diff
@@ -0,0 +1,12 @@
+Name:Bane's Contingency
+ManaCost:1 U U
+Types:Instant
+A:SP$ Pump | IsCurse$ True | TargetType$ Spell | TgtZone$ Stack | TgtPrompt$ Select target spell | ValidTgts$ Card | SubAbility$ DBBranch | StackDescription$ SpellDescription | SpellDescription$ Counter target spell. If that spell targets a commander you control, instead counter that spell, scry 2, then draw a card.
+SVar:DBBranch:DB$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ EQ1 | TrueSubAbility$ DBCounterScry | FalseSubAbility$ DBCounter | StackDescription$ None
+SVar:DBCounterScry:DB$ Counter | Defined$ Targeted | SubAbility$ DBScry
+SVar:DBScry:DB$ Scry | ScryNum$ 2 | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | NumCards$ 1
+SVar:DBCounter:DB$ Counter | Defined$ Targeted
+SVar:X:Count$Compare CheckTgt GE1.1.0
+SVar:CheckTgt:TargetedByTarget$Valid Card.IsCommander+YouCtrl
```

Comment:
Check if it needs inZoneBattlefield?
> Notably, if a player uses Bane's Contingency to counter a spell that would counter their commander, they do not scry or draw a card.

Reply:
Added the +inZoneBattlefield clause to CheckTgt. Thanks for catching that!